share/appliance_cli/src/sandbox.py

```diff
@@ -40,6 +55,10 @@ def _prompt_kwargs():
         'get_title': lambda: title
     }
 
+def toggle_advanced_mode(switch):
+    environ['ADVANCED'] = str(switch)
+    print('Advanced mode %s' % ('enabled' if switch else 'disabled'))
+    execve(argv[0], argv, environ)
```

Comment:
Minor thing but could be worth a comment about what we're doing here as I can imagine this looking unusual to someone not in our recent discussions. Basically just a brief explanation of why we want to `exec` and why we need to pass both `argv[0]` and `argv`.